src/de/aniworld/src/eu/kanade/tachiyomi/animeextension/de/aniworld/AniWorld.kt

```diff
@@ -41,13 +36,10 @@ import uy.kohesive.injekt.injectLazy
 
 class AniWorld : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
-    override val name = "AniWorld (experimental)"
+    override val name = "AniWorld"
```

Comment:
Keep the old `id` since the name was changed, so that users won't need to migrate